### PR TITLE
Add Priyal Gupta to Contributors list

### PR DESCRIPTION
**Pull request by priyalgupta09**

### CODE REVIEW

File: Strings/leftTrim.c [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+q++;
+p++;
+}
+*p='\0';
+}
+}
+
+int main()
+{
+char a[37];
```
Comment (SidJain1412) [inline Strings/leftTrim.c:35]: is there any reason for this constraint? in all 3 programs

Comment (priyalgupta09) [reply Strings/leftTrim.c:35]: no. you can take the size of string according to your requirement.

Comment (vishal-burman) [reply Strings/leftTrim.c:35]: @SidJain1412 As far as my experience goes in Codeforces and AtCoder upper limits in array constraints are usually taken as Integer.MAX_VALUE in Java and array_type[1234567] for c, c++. The program could accommodate those constraints if it's helpful.

Comment (SidJain1412) [reply Strings/leftTrim.c:35]: @priyalgupta09 I agree with @vishal-burman , please modify to remove this constraint. If the string is more than 37 chars then the program won't work right? no reason for this constraint as far as i can see

Comment (priyalgupta09) [reply Strings/leftTrim.c:35]: okay i will make chnge.

Comment (priyalgupta09) [reply Strings/leftTrim.c:35]: @SidJain1412  I have made changes and sent a pull request gin please see and accept my pull request

**Pull request closed without merging**